cultcargo/genesis/wsclean/__init__.py

```diff
@@ -48,6 +48,7 @@ def make_stimela_schema(params: Dict[str, Any], inputs: Dict[str, Parameter], ou
     multitime = params.get('multi.interval', False) or not isinstance(ntime, int) or ntime > 1
 
     for imagetype in "dirty", "restored", "residual", "model":
+        if params.get("make-psf-only", False): continue # skip all clean output
```

Comment:
Why not move the conditional outside the loop? I.e. ``if not params.get("make-psf-only", False)``, then do the loop? No point looping and checking when it's invariant inside the loop.